test/integration/pods/pods_test.go

```diff
@@ -37,7 +37,9 @@ func TestPodUpdateActiveDeadlineSeconds(t *testing.T) {
 	ns := framework.CreateTestingNamespace("pod-activedeadline-update", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
 
-	client := client.NewOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
Please stop spamming me with email updates about this, The mute link does not work. Someone incorrectly mentioned me in some comment now I get 20-30 email spam messages a day from this thread. Someone remove me.